rapthor/lib/image.py

```diff
@@ -157,7 +157,8 @@ def calc_noise(self, niter=1000, eps=None):
         """
         if eps is None:
             eps = np.nanstd(self.img_data)*1e-3
-        data = self.img_data[ ~np.isnan(self.img_data) ]  # remove nans
+        data = self.img_data[::4]  # sample every forth pixel
```

Comment:
Using only a quarter of the data comes out of the blue, but there's probably a sensible argument. Should this subsampling ratio be settable?

Reply:
Good point -- it is just used to speed up the noise calculation. Probably there is some ideal value, given normal sampling (~ 3 pixels per beam), but I found 4 worked well in practice. I'll add the option to set it and some more comments on its purpose.